enterprise/core-edge/src/test/java/org/neo4j/coreedge/edge/EdgeStartupProcessTest.java

```diff
@@ -109,9 +109,9 @@ public void shouldNotStartWithMismatchedNonEmptyStore() throws Throwable
             edgeStartupProcess.start();
             fail( "should have thrown" );
         }
-        catch ( IllegalStateException ex )
+        catch ( Exception ex )
```

Comment:
The tricky part with expecting Exception is that it can be thrown from anywhere in the code. I think that either the code should not throw Exception in start() (introduce a new type, perhaps) or the catch should assert on the exception message.